workos/utils/request.py

```diff
@@ -95,11 +100,11 @@ def request(
                 raise AuthenticationException(response)
             elif status_code == 403:
                 raise AuthorizationException(response)
-            elif status_code == 404:
```

Comment:
Why remove this?

Reply:
Definitely didn't mean to remove this, fat finger error. Will update.

Reply:
@PaulAsjes updated